PWGLF/Tasks/Strangeness/derivedcascadeanalysis.cxx

```diff
@@ -61,6 +61,10 @@ using namespace o2::framework;
 using namespace o2::framework::expressions;
 using std::array;
 
+// simple checkers
+#define BIT_SET(var, nbit) ((var) |= (1 << (nbit)))
+#define BIT_CHECK(var, nbit) ((var) & (1 << (nbit)))
```

Comment:
Why don't you use existing macros `SETBIT`, `TESTBIT`?

Reply:
I changed it.